ai-service/src/ai_service/embeddings/transformer.py

```
@@ -25,32 +28,36 @@
-def get_model(
-    trust_remote_code: bool = False,  # Might be required for some code models
-) -> SentenceTransformer:
+def initialize_model(trust_remote_code: bool = False) -> None:
+    """Initialize the embedding model at application startup."""
+    global _model
+    if _model is None:
+        model_name = utils.get_env_var(constants.EMBEDDING_MODEL)
+        device = _get_device()
+
+        try:
+            logger.info(f"Device: {device}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)